### PR TITLE
Redesign of the fellowship website

### PR DESCRIPTION
**Pull request by designmatty**
### Overview
- This PR completely wipes away the default jekyll theme in favor of a custom theme
- Implements working mailchimp forms for job/fellowship notifications (tested and confirmed working)
- Copy has been imported from the current fellowships.azavea.com
- new settings in `_config.yml` 
  - `fellowship_in_session` will dynamically change certain areas of the site for easy maintenance
  - `fellowship_timeframe` will change all references to time throughout the site
- new projects collection
- adds jekyll-avatar plugin for mentors section (works with github pages)

#### TODO in future PR's
The following will need issues made once this PR is approved and merged.
- add imagery throughout site
- implement image carousels on homepage from designs
  - [see about and qualifications carousels](https://share.goabstract.com/4e74311c-25ac-4aab-b976-fb7fcd3535c0)
- continue copy edits
  - new content sections will be added as needed, but will need help from a designer
  - Dan has already added some content in frontmatter which will be added to the theme files in future PR's
- update projects and mentors
- add google hire apply link
- favicons and seo
- fix broken links
- style 404 page
- add Project meta details into individual project posts
  - ie: requirements, difficulty, tags

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      <div class="column footer-social">
+        <a href="http://twitter.com/azavea"><img src="{{ site.baseurl }}/assets/icons/icon-twitter.svg" alt="Azavea twitter"></img></a>
+        <a href="http://facebook.com/azavea"><img src="{{ site.baseurl }}/assets/icons/icon-facebook.svg" alt="Azavea facebook"></img></a>
+        <a href="http://www.linkedin.com/company/azavea"><img src="{{ site.baseurl }}/assets/icons/icon-linkedin.svg" alt="Azavea linkedin"></img></a>
+        <a href="http://github.com/azavea"><img src="{{ site.baseurl }}/assets/icons/icon-github.svg" alt="Azavea github"></img></a>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column footer-bottom">
+        <a href="http://azavea.com/terms-of-use/">Terms of Use</a> •
```
Comment (hectcastro) [inline _includes/footer.html]: Nitpick, but we can make these HTTPS to avoid a redirect for whoever clicks.

File: _projects/cloud-detection.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (hectcastro) [inline _projects/cloud-detection.md]: This file name contains a dash instead of an underscore (like the instructions above state).

**Comment by hectcastro** [comment]
screen shot 2018-01-04 at 17 26 21

File: _projects/cloud-detection.md (hunk 2 above)
Comment (designmatty) [reply _projects/cloud-detection.md]: good catch.

File: _includes/footer.html (hunk 1 above)
Comment (designmatty) [reply _includes/footer.html]: Will do

**Pull request merged**